federatedscope/main.py

```diff
@@ -34,8 +35,13 @@
 
     init_cfg.freeze()
 
+    # allow different settings for different clients
+    # cfg_client.merge_from_file(args.cfg_client)
+    cfg_client = CfgNode.load_cfg(open(args.cfg_client, 'r'))
```

Comment:
We should consider the situation that `args.cfg_client=None` or invalid path.

Reply:
It is modifed accordingly